content/docs/operating/configuration.md

```diff
@@ -621,7 +621,12 @@ tls_config:
 Where `<role>` must be `endpoints`, `service`, `pod`, or `node`.
 
 See [this example Prometheus configuration file](https://github.com/prometheus/prometheus/blob/master/documentation/examples/prometheus-kubernetes.yml)
-for a detailed example of configuring Prometheus for Kubernetes.
+for a detailed example of configuring Prometheus for Kubernetes. Also check out
+the [Prometheus Operator](https://github.com/coreos/prometheus-operator) and
+[kube-prometheus](https://github.com/coreos/kube-prometheus) when using
+Prometheus with Kubernetes. It automates your Prometheus setup on top of
+Kubernetes and does not require you to learn the configuration idioms of
+Prometheus.
```

Comment:
Even more concise. However, https://github.com/coreos/kube-prometheus links to https://github.com/coreos/prometheus-operator , but not vice versa. So if we want to give only one pointer, it should be to https://github.com/coreos/kube-prometheus.